src/main/java/appeng/tile/inventory/AppEngInternalAEInventory.java

```diff
@@ -131,9 +117,14 @@ private void readFromNBT( final NBTTagCompound target )
 			}
 		}
 	}
+	
+    protected int getStackLimit( int slot, @Nonnull ItemStack stack )
+    {
+        return Math.min( getSlotLimit(slot), stack.getMaxStackSize() );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)